git_repo/services/ext/github.py

```diff
@@ -277,13 +277,14 @@ def request_create(self, onto_user, onto_repo, from_branch, onto_branch, title=N
                     base=onto_branch,
                     body=description)
 
-            return {
-                'local': from_branch,
-                'project': '/'.join([onto_user, onto_repo]),
-                'remote': onto_branch,
-                'ref': request.number,
-                'url': request.html_url
-            }
+            yield 'Successfully created request of `{local}` onto `{project}:{remote}, with id `{ref}'.format(
+                local=from_branch,
+                project='/'.join([onto_user, onto_repo]),
+                remote=onto_branch,
+                ref=request.number
+            )
+            yield 'available at {}'
+            yield request.html_url
```

Comment:
There you've changed the way `request_create` works across the tool, but as you're overloading `action_create` in gerrit, you did not update the overloaded method, and now all `request_create` tests fail! ☹